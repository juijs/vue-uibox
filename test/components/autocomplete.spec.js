import { mount } from 'vue-test-utils'
import UixAutocomplete from '@/components/autocomplete'

describe('autocomplete.js', () => {
    const wrapper = mount(UixAutocomplete, {
        propsData: {
            width: 200,
            items: [
                "ActionScript",
                "AppleScript",
                "Asp",
                "BASIC",
                "C",
                "C++",
                "Clojure",
                "COBOL",
                "ColdFusion",
                "Erlang",
                "Fortran",
                "Groovy",
                "Haskell",
                "Java",
                "JavaScript",
                "Lisp",
                "Perl",
                "PHP",
                "Python",
                "Ruby",
                "Scala",
                "Scheme"
            ],
            value: ""
        }
    });

    it('snapshot!!', () => {
        wrapper.vm.$nextTick(() => {
            expect(wrapper.vm.$el).toMatchSnapshot();
        });
    });
})