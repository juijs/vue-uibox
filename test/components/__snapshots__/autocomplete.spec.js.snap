// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`autocomplete.js snapshot!! 1`] = `
<input
  class="input small"
  style="width: 200px;"
  type="text"
/>
`;
